docs/cluster-setup-guide/deploy-cluster/sysadmin-deploy-on-k8s/custom-pod-specs.rst

```diff
@@ -50,9 +50,13 @@ This section describes which fields users can and cannot configure when specifyi
 Determined does not currently support configuring:
 
 -  Pod Name - Determined automatically assigns a name for every pod that is created.
--  Pod Namespace - Determined launches all tasks in the Namespace in which the Determined master is
-   running.
+
+-  Pod Namespace - Determined launches all tasks in the Namespace coupled to the resource pool in
```

Comment:
I was a little confused reading this the first time. Maybe something just like:

`Determined automatically sets the pod namespace based on the resource pool the task belongs to. This can be done...`